neo/tools/debugger/DebuggerScript.cpp

```diff
@@ -91,7 +87,7 @@ appropriate for the file being loaded.  If the script cant be compiled
 the loading of the script fails
 ================
 */
-bool rvDebuggerScript::Load ( const char* filename )
+bool rvDebuggerScript::Load ( const char* filename)
```

Comment:
TBH removing the space before `)` doesn't look that nice, esp. when leaving the space after `(` ;)

Reply:
yes will fix that.
If you have seen more whitespace bugs, let me know!